crypto/pkcs12/p12_crt.c

```diff
@@ -208,13 +208,12 @@ int PKCS12_add_safe(STACK_OF(PKCS7) **psafes, STACK_OF(PKCS12_SAFEBAG) *bags,
     PKCS7 *p7 = NULL;
     int free_safes = 0;
 
-    if (!*psafes) {
+    if (*psafes == NULL) {
         *psafes = sk_PKCS7_new_null();
-        if (!*psafes)
+        if (*psafes == NULL)
             return 0;
         free_safes = 1;
-    } else
-        free_safes = 0;
+    }
```

Comment:
Why is the else removed?

Reply:
I assume it's because it's already initialized to 0

Reply:
Exactly, it's initialized in the declaration.